### DISCUSSION TITLE
Is it possible to deploy a flet app to the google app engine standard environment

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#1084] [Q&A]
I ask this because in the flexible environment there is no free tier.

**Comment by iron3oxide** [comment]
From what I read, it should be possible as GAE [supports WebSockets](https://cloud.google.com/appengine/docs/flexible/using-websockets-and-session-affinity?tab=python). You'd have to set a custom entrypoint (`flet run main.py` or something alike) and try your luck.

**Reply by FeodorFitsner** [reply]
Previous discussion with you: https://github.com/flet-dev/flet/discussions/512

**Reply by iron3oxide** [reply]
I think the OP wants to know if they can deploy to the standard environment too. Or does the discussion link mean that it's not possible? It says "in general you have to deploy to the flexible environment", which might mean that the standard environment can work too, depending on interpretation.

**Reply by FeodorFitsner** [reply]
Ah, right, standard! Didn't notice that, sorry. I didn't realize you can try using [GAE standard python](https://cloud.google.com/appengine/docs/python) environment.
